### PR TITLE
Joacarrilloco patch 1

### PR DESCRIPTION
**Pull request by joacarrilloco**

### CODE REVIEW

File: Utilities/test.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+g++ -std=c++17 gen.cpp -o gen
```
Comment (joacarrilloco) [inline Utilities/test.sh:1]: Compilation command included KISS

File: Utilities/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+compile:
+	g++ -std=c++17 sol.cpp -static -Wall -Wno-unused-result -O2 -o sol
+compileAndRun:
+	g++ -std=c++17 sol.cpp -static -Wall -Wno-unused-result -O2 -o sol && sol < in
```
Comment (osdajigu) [inline Utilities/Makefile]:
```suggestion
	g++ -std=c++17 sol.cpp -static -Wall -Wno-unused-result -O2 -o sol && ./sol < in
```

File: Utilities/Makefile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+compile:
+	g++ -std=c++17 sol.cpp -static -Wall -Wno-unused-result -O2 -o sol
```
Comment (osdajigu) [inline Utilities/Makefile]:
```suggestion
	g++ -std=c++17 sol.cpp -static -Wall -Wshadow -Wno-unused-result -O2 -o sol
```

File: Utilities/Makefile [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+compileAndRun:
+	g++ -std=c++17 sol.cpp -static -Wall -Wno-unused-result -O2 -o sol && ./sol < in
```
Comment (osdajigu) [inline Utilities/Makefile]:
```suggestion
	g++ -std=c++17 sol.cpp -static -Wall -Wshadow -Wno-unused-result -O2 -o sol && ./sol < in
```

**Pull request merged**